File: src/Generated20260216.go
package main

import "fmt"

type Generated20260216 struct {
    // Generated struct 20260216_221102_865035
    Value int
    Created string
}

func (c *Generated20260216) Process() int {
    result := c.Value
    for i := 0; i < 31; i++ {
        result += i * 7
    }
    return result
}

func main() {
    obj := Generated20260216{Value: 207, Created: "20260216_221102_865035"}
    fmt.Println(obj.Process())
}

File: src/Generated20260216.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260216 {
public:
    // Generated class 20260216_220536_856007
    int value = 654;
    string created = "20260216_220536_856007";
    
    int process() {
        int result = value;
        for (int i = 0; i < 32; i++) {
            result += i * 2;
        }
        return result;
    }
};

int main() {
    Generated20260216 obj;
    cout << obj.process() << endl;
    return 0;
}